JavaScript (v3): Topics and Queues workflow (#5214)
awsdocs/aws-doc-sdk-examples: javascriptv3/example_code/cross-services/wkflw-topics-queues/README.md

The sample code builds a command line application that asks you for input. This is implemented in multiple programming languages, and the interface can vary slightly between languages. This folder contains the JavaScript implementation.

## ⚠️ Important

- Running this code might result in charges to your AWS account.
- Running the tests might result in charges to your AWS account.
- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
- This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).

## Prerequisites

- NodeJS

## Instructions

**Note**: All code examples are written in ECMAscript 6 (ES6).

1. Install dependencies. `npm i`
2. Run the example. `node index.js`

> **Note**: By default the prompts in the example will print slowly for readability. This can be disabled by passing the `--no-logger-delay` option.

Reviewer: ```suggestion
**Note**: By default the prompts in the example will print slowly for readability. To disable this, pass the `--no-logger-delay` option.
```
